app/Models.txt

```diff
@@ -1,44 +0,0 @@
-A List of models and relationships
```

Comment:
Delete rather than update is deliberate?

Reply:
yep, totally out of date